Lawgical-Immigration/Lawgical
# Miri/document scraping

### Description

This PR merges functionality to scrape the uploaded paystub and fill out the I-129 form. 
- The code for parsing/filling out I-129 is in `server/controllers/documentController`
- Updated gitignore


## To test
- run `npm run start`
- using Postman, send a POST request to `localhost:5050/document/[random characters]` with a paystub attached to the request
-  sending the request should generate a new pdf file in `server/i-129-sample` under the name **[first name]_[last name]_i-129.pdf**

Note: The video is too large to upload to the PR so has been sent to AK via email. Please contact her for the video.
